tests/steps/steps.py

```diff
@@ -1,28 +1,57 @@
-from behave import given, when, then
 import base64
-from algosdk import kmd
-from algosdk.future import transaction
-from algosdk import encoding
-from algosdk import algod
-from algosdk import account
-from algosdk import mnemonic
-from algosdk import wallet
-from algosdk import auction
-from algosdk import util
-from algosdk import constants
-from algosdk import logic
-from algosdk.future import template
 import os
+import random
 from datetime import datetime
-import hashlib
 
+from algosdk import (
+    account,
+    algod,
+    auction,
+    constants,
+    encoding,
+    kmd,
+    logic,
+    mnemonic,
+    util,
+    wallet,
+)
+from algosdk.future import transaction
+from behave import given, then, when
 from nacl.signing import SigningKey
 
 token = "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
 algod_port = 60000
 kmd_port = 60001
 
 
+# Initialize dev mode accounts.
+def initialize_dev_mode_account(context):
+    context.dev_sk, context.dev_pk = account.generate_account()
+    payment = transaction.PaymentTxn(
+        sender=context.accounts[0],
+        sp=context.acl.suggested_params_as_object(),
+        receiver=context.dev_pk,
+        amt=10_000_000,
```

Comment:
is 10 algo's enough? I'm not sure how much the account is supposed to spend during the lifetime of all tests, but it's not hard to imagine it could run out of algos if it's workload is sufficiently complex. Might be worth creating a constant `ACCOUNT_INITIAL_MICROALGOS: int = ... the amount...` at the top of the file and use that here.

Reply:
For the v1 steps, I thought it would be sufficient to hardcode to a smaller value because the workload isn't too complex and we probably wouldn't be extending the tests anyway - but to be extra safe, I set the receiver to itself so it wouldn't run out of algos. I do like the constant suggestion and implemented in the latest commit.